src/styles/hero.scss

```diff
@@ -11,18 +11,28 @@ body,
 
 .hero,
 .hero-wrapper {
-	height: 100%;
+	// height: 100%;
```

Comment:
Remove this commented line and any other commented lines if they are unused code. But it's ok to keep commented lines if it's an instruction, todo, reminder, etc

Reply:
Removed excess comments